pkg/internal/testhelpers/postgres_container.go

```diff
@@ -175,11 +175,6 @@ func (e ExtensionState) GetDockerImageName() (string, error) {
 		image = "timescale/timescaledb:latest-" + PGTag
 	case Timescale2AndPromscale, MultinodeAndPromscale:
 		image = LatestDBHAPromscaleImageBase + ":" + PGTag + "-latest"
-
-		//TODO: remove split once pg14 ha image published
```

Comment:
There is another todo like this to be removed in `end_to_end_tests/continuous_aggs_test.go`. Should that be removed as well?

Reply:
Do you have a line reference? I could not find it.

Reply:
https://github.com/timescale/promscale/pull/960/files#diff-48513ed7faf5b2c9cf4e5c4226fe3366eeb5d01ff0b865de28fbf64482340815L464-L467